Resources/Prototypes/secret_weights.yml

```diff
@@ -1,8 +1,30 @@
 - type: weightedRandom
   id: Secret
-  weights:
+  weights: #these are the presets!!! NOT gamerules.
     Nukeops: 0.20
     Traitor: 0.50
     Zombie: 0.10
     Revolutionary: 0.20
-    
\ No newline at end of file
+
+- type: weightedRandom
+  id: Secret_MRP
+  weights:
+    Nukeops: 0.15
+    Traitor: 0.40
+    Zombie: 0.10
+    Revolutionary: 0.10
+    Extended: 0.25
+
+- type: weightedRandom
+  id: Secret_LRP
+  weights:
+    Nukeops: 0.20
+    Traitor: 0.50
+    Zombie: 0.10
+    Revolutionary: 0.15
+    Extended: 0.05
+
+- type: weightedRandom
+  id: Secret_Extended
+  weights:
+    Extended: 1.00
```

Comment:
Admins can force extended. No need for another "secret extended"

Reply:
There are cases in which having extended be forced but not obvious is desired, and there's no downside to having it.